internal/bminventory/inventory.go

```diff
@@ -688,11 +693,33 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 		return nil, common.NewApiError(http.StatusInternalServerError, err)
 	}
 
+	if b.ocmClient != nil && b.Config.WithAMSSubscriptions {
+		if err := b.integrateWithAMS(ctx, *cluster.ID); err != nil {
+			log.WithError(err).Errorf("Cluster %s failed to integrat with AMS", id)
```

Comment:
s/integrat/integrate
or be more specific, e.g. failed to manipulate subscription

Reply:
Fixed, I don't want to change the message content because we will have additional logs inside the called function for each one of the failures, this one was "generic" in purpose to not duplicates the error messages.